installer/pyinstaller/build-linux.sh

```diff
@@ -2,6 +2,7 @@
 binary_zip_filename=$1
 python_library_zip_filename=$2
 python_version=$3
+nightly_build=$4
```

Comment:
Why is this here?

Reply:
The this due to when I pulled from the feature branch, my branch was already ahead of the feature branch. So it included extra commits. Not sure if any good solutions here. Will discuss about this as a part of the release improvements.